### ISSUE TITLE
CLI should use a non-zero exit code if rules are outdated

### ISSUE THREAD

**Issue by thibaudcolas** [Donov4n/stylelint-find-new-rules#4]
At the moment the CLI stops with an exit code of 0 regardless of what outdated / invalid / unused rules it found.

I would like to use this tool so I know exactly what rules have been introduced in the selection of plugins I'm using, and what might be changing in plugins. It would be nice to be able to do this in CI, so the build of my config breaks if an outdated / invalid rule is used – a non-zero exit code would be a nice way to do that.

In the meantime I'm manually checking the output from the tool:

```
stylelint-find-new-rules src/unused.js --unused --deprecated --invalid | tee /dev/tty | grep 'All rules are up-to-date!' -c -q
```

This works, but feels unnecessarily complex.

**Comment by Donov4n** [comment]
Thank you for the request, this will be available in the next "patch" version (`1.0.2`).

**Comment by thibaudcolas** [comment]
Nice! Well that was fast :)
